contracts/tasks/account.js

```diff
@@ -190,19 +193,26 @@ async function mint(taskArguments, hre) {
     throw new Error("Task can only be used on local or fork");
   }
 
-  const ousdProxy = await ethers.getContract("OUSDProxy");
-  const ousd = await ethers.getContractAt("OUSD", ousdProxy.address);
```

Comment:
Shouldn't this correctly use the mainnet address?

Reply:
I might be doing something wrong, but hardhat isn't able to find the already deployed contracts for me.
![Screenshot 2021-10-05 at 10 17 52](https://user-images.githubusercontent.com/579910/135988613-5d418717-dc5e-4dbc-9708-02e066441671.png)

Even though the `deployments/localhost` gets copied over in `node.sh` from the `deployments/mainnet` the `localhost` folder is immediately deleted when the `FORK=true npx --no-install hardhat node --export...` is ran.

It would be super useful to make fork already source the deployed mainnet contracts from deployments folder. Is this not happening to you?

To test it these 2 commands can be ran: 
```
FORK=true npx hardhat fund --index 4 --num 1 --amount 1000000 --network localhost
FORK=true npx hardhat mint --index 4 --num 1 --amount 50000 --network localhost
```

Reply:
Ahh yea the behaviour of hardhat changed and we upgraded. Fix was here https://github.com/OriginProtocol/origin-dollar/pull/765

Reply:
Thanks, was wondering why that reset was happening... Works like a charm now. 

Have undid the unnecessary code changes. Can you please take another look?